gradle.properties

```diff
@@ -14,4 +14,5 @@ kotlin.mpp.androidSourceSetLayoutVersion=2
 android.useAndroidX=true
 android.compileSdk=33
 android.targetSdk=33
-android.minSdk=21
\ No newline at end of file
+android.minSdk=21
+android.defaults.buildfeatures.buildconfig=true
```

Comment:
could you explain the flag?

Reply:
For some reason, BuildConfig was not being created or picked up. This flag helped.

Reply:
https://stackoverflow.com/a/74692390

Reply:
Thanks